src/components/Router/StepsRouter.js

```diff
@@ -28,7 +28,10 @@ class StepsRouter extends Component {
     return (
       //TODO: Wrap CurrentComponent in themeWrap HOC
       <div
-        className={classNames(theme.step, { [theme.fullScreenStep]: isFullScreen, [theme.noLogo]: hideOnfidoLogo })}
+        className={classNames(theme.step, {
+          [theme.fullScreenStep]: isFullScreen,
+          [theme.noLogo]: globalUserOptions.enterpriseFeatures?.hideOnfidoLogo && hideOnfidoLogo 
```

Comment:
Minor: You could assign this to a variable to make it more readable. You could call it something like `const shouldHideLogo = globalUserOptions.enterpriseFeatures?.hideOnfidoLogo && hideOnfidoLogo `

Reply:
Ignore this comment, I added on an older commit